### PR TITLE
here are the queries for the resolvers

### PR DESCRIPTION
**Pull request by lsoth**

### CODE REVIEW

File: server/schemas/resolverRoutes/queries.js [hunk 1]
```diff
@@ -1,13 +1,43 @@
...
+    if (context.user) {
+      return User.findOne({ _id: context.user._id });
+      }
+      throw new AuthenticationError('You need to be logged in!');
+  },
+  brand: async ( parent , { _id }) => {
+    return await Brand.findById(_id).populate('skateVideos')
+  },
+  brands: async () => {
+    return await Brand.find().populate('skateVideos');
```
Comment (delizoderek) [inline server/schemas/resolverRoutes/queries.js:16]: Don't you need to send in an empty object with these queries?

File: server/schemas/resolverRoutes/queries.js [hunk 2]
```diff
@@ -1,13 +1,43 @@
...
+  },
+  skaters: async () => {
+    return await Skater.find();
+  },
+  skater: async (parent, { _id }) => {
+    return await Skater.findById(_id);
+  },
+  skateVideos: async (parent, { skater, name }) => {
```
Comment (delizoderek) [inline server/schemas/resolverRoutes/queries.js:24]: Why do you need to the skater and name for querying skate videos? The brands and skaters have a reference to the videos. Don't you just need to populate(Skater) populate(SkateVideo)

**Review by delizoderek — CHANGES_REQUESTED** [review]

File: server/schemas/resolverRoutes/queries.js (hunk 1 above)
Comment (lsoth) [reply server/schemas/resolverRoutes/queries.js:16]: I was looking through examples and I do not believe you need an empty object

File: server/schemas/resolverRoutes/queries.js (hunk 2 above)
Comment (lsoth) [reply server/schemas/resolverRoutes/queries.js:24]: I just did that so you can query skate videos for specific skaters, or the name of the video, its similar to how its set up in the  Redux Code review app, I can update the typedefs to change it so we do not need to require the skaters or name.

File: server/schemas/resolverRoutes/queries.js (hunk 1 above)
Comment (delizoderek) [reply server/schemas/resolverRoutes/queries.js:16]: Ok

**Review by delizoderek — APPROVED** [review]

**Pull request merged**